inc/saibridge.h

```diff
@@ -118,12 +131,21 @@ typedef enum _sai_bridge_port_attr_t
     SAI_BRIDGE_PORT_ATTR_PORT_ID,
 
     /**
+     * @brief Tagging mode of the bridge port
+     *
+     * @type sai_bridge_port_tagging_mode_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @condition SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_SUB_PORT
+     */
+    SAI_BRIDGE_PORT_ATTR_TAGGING_MODE,
+
+    /**
      * @brief Associated Vlan
      *
      * @type sai_uint16_t
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
      * @isvlan true
-     * @condition SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_SUB_PORT
+     * @condition SAI_BRIDGE_PORT_ATTR_TAGGING_MODE == SAI_BRIDGE_PORT_TAGGING_MODE_TAGGED
```

Comment:
also one conditional attribute depends on another conditional attribute, this is not supported now